gpkit/nomials/data.py

```diff
@@ -66,27 +72,6 @@ def init_from_nomials(self, nomials):
     def __repr__(self):
```

Comment:
a bit above this point -- can we get rid of `self.nomials` being defined outside `__init__` now?